content/en/docs/contribute/advanced.md

```diff
@@ -2,7 +2,7 @@
 title: Advanced contributing
 slug: advanced
 content_template: templates/concept
-weight: 30
+weight: 98
```

Comment:
nit: You could created a separate PR for the extra space changes.
nit: I think it is suggested to use whole numbers for the weight value.

Reply:
Fix links to pages that are no longer available.